iosApp/iosApp/ViewModels/SettingsViewModel.swift

```diff
@@ -61,14 +56,13 @@ class SettingsViewModel: ObservableObject {
                         )
                     ),
                 ]),
-                MoreSection(
-                    id: .settings,
-                    items: settings.filter { $0.category == .settings }.map { MoreItem.toggle(setting: $0) }
-                ),
-                MoreSection(
-                    id: .featureFlags,
-                    items: settings.filter { $0.category == .featureFlags }.map { MoreItem.toggle(setting: $0) }
-                ),
+                MoreSection(id: .settings, items: [
+                    .toggle(setting: .hideMaps, value: settings[.hideMaps] ?? false),
+                ]),
+                MoreSection(id: .featureFlags, items: [
+                    .toggle(setting: .map, value: settings[.map] ?? false),
+                    .toggle(setting: .searchRouteResults, value: settings[.searchRouteResults] ?? false),
+                ]),
```

Comment:
The settings get bridged as an unordered dictionary, meaning these toggle switches weren't in a deterministic order when I was keeping `.filter().map()`, and I figured that the simplest way to ensure they have a deterministic order is to just hardcode that order.